### PR TITLE
Different concrete floor

### PR DESCRIPTION
**Pull request by OromisElf**
#### Summary
Content "a player's constructed concrete floor is a different one"

#### Purpose of change

during the making of #47919 I noticed that the concrete below a metal gate and the concrete floor a player can build are different tiles so I am just making this pr to see if I can change that

#### Describe the solution

I changed the tile a player makes when constructing a concrete floor (as well as construction requirements)

#### Describe alternatives you've considered

not doing that

#### Testing

started a game, constructed a concrete floor, constructed a few things that need a concrete floor and confirmed they can still be built

#### Additional Context

the mentioned pr doesn't use concrete floor anymore but I'm not gonna close this issue yet because I find the concrete floor I used prettier

### CODE REVIEW

File: data/json/construction.json [hunk 1]
```diff
@@ -931,7 +931,7 @@
     "qualities": [ [ { "id": "SMOOTH", "level": 1 } ] ],
     "components": [ [ [ "concrete", 2 ] ], [ [ "water", 2 ] ] ],
     "pre_terrain": "t_pit_shallow",
-    "post_terrain": "t_concrete"
+    "post_terrain": "t_thconc_floor"
```
Comment (Maleclypse) [inline data/json/construction.json:934]: This floor comes with a roof.  Is that your intention?   Would some kind of mechanical device need to sit on top of the roof in this case?

**Comment by actual-nh** [comment]
Fix conflicts, please?

**Comment by Maleclypse** [comment]
> Fix conflicts, please?

This probably needs a rebase to fix it,

File: data/json/construction.json (hunk 1 above)
Comment (OromisElf) [reply data/json/construction.json:934]: sorry for long inactivity but no that was not my intention
I really just wanted to make the concrete floor the player can make, the same that appears under a metal gate

**Comment by ZhilkinSerg** [comment]
Feel free to resolve conflicts and ping me to reopen (or open a new PR).

**Pull request closed without merging**